CMakeLists.txt

```diff
@@ -793,14 +793,21 @@ INSTALL(DIRECTORY include/ ${CMAKE_BINARY_DIR}/include/
   FILES_MATCHING PATTERN "*.h")
 
 # examples:
-INSTALL(DIRECTORY cookbooks/
-  DESTINATION cookbooks
-  COMPONENT examples
-  FILES_MATCHING PATTERN "*")
-INSTALL(DIRECTORY benchmarks/
-  DESTINATION benchmarks
-  COMPONENT examples
-  FILES_MATCHING PATTERN "*")
+SET(ASPECT_INSTALL_EXAMPLES OFF CACHE BOOL "If ON all cookbooks and benchmarks will be built and installed.")
```

Comment:
Should we call this ASPECT_ENABLE_EXAMPLES instead? (it involves building and installing)

The current name is ok as well, so I leave this up to you.

Reply:
I think I prefer the current name. ENABLE implies to me that I cannot use the cookbooks and benchmarks if it is turned off, but the folders will always be there in the source directory. And building the examples without installing is actually not very helpful, because the libraries will be in a sub-directory of the build directory, where the .prm files will not find them.